Source/WebCore/html/ImageBitmap.cpp

```diff
@@ -981,6 +981,14 @@ RefPtr<ImageBuffer> ImageBitmap::takeImageBuffer()
     return std::exchange(m_bitmap, nullptr);
 }
 
+RefPtr<NativeImage> ImageBitmap::copyNativeImage()
+{
+    if (!m_bitmap)
+        return nullptr;
+    return m_bitmap->copyNativeImage();
```

Comment:
One liner maybe?
Also `m_bitmap` is not const, so we likely want to ref it before calling copyNativeImage.